src/main/scala/org/constellation/snapshotstreaming/FileBasedLastGlobalSnapshotStorage.scala

```diff
@@ -0,0 +1,82 @@
+package org.constellation.snapshotstreaming
+
+import java.nio.file.NoSuchFileException
+import cats.effect.Async
+import cats.effect.std.Semaphore
+import cats.syntax.applicativeError._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
```

Comment:
do we have a scalafmt/fix here, like in tessellation?

Reply:
We have something but it seems not completely configured. I wanted to create a ticket to configure it as a separate task and not do it here to not mix issues in one PR.